requirements-dev-frozen.txt

```diff
@@ -146,9 +146,9 @@ rfc3986-validator==0.1.1
 rsa==4.8
 s3transfer==0.5.2
 selenium==4.3.0
-sentry-arroyo==1.0.4
+sentry-arroyo==1.0.5
 sentry-relay==0.8.13
-sentry-sdk==1.9.8
+sentry-sdk==1.9.10
```

Comment:
concernign that it did that. please check with dev infra team why it did that and revert this line

Reply:
this is just a weird artifact of how github displays diffs -- it's already at that version on the primary branch